src/components/message/message.scss

```diff
@@ -0,0 +1,57 @@
+@use "../../design-tokens/colors.yml";
+@use "../../mixins/focus";
+@use "../../design-tokens/font.yml";
+@use "../../design-tokens/sizes.yml";
+@use "../../mixins/ms";
+@use "../../mixins/unit";
+
+/**
+ * 1. Display message and close button in a centered row.
+ * 2. Push message and close button as far apart as possible.
+ * 3. Condense line height in case it breaks to multiple lines.
+ *
+ * @TODO
+ * 1. Style `:focus` state
+ */
+
+.c-message {
+  align-items: center; /* 1 */
+  background-color: colors.$gray-lighter;
+  color: colors.$text-dark;
+  display: flex; /* 1, 2 */
+  justify-content: space-between; /* 2 */
+  line-height: ms.step(1, 1); /* 3 */
+  padding: ms.step(1) ms.step(0);
+}
+
+.c-message--withClose {
+  padding-right: ms.step(0);
+}
+
+/**
+ * 1. Button background-color should inherit the same color as its parent.
+ * 3. Close icon should be the same color as the parent text color.
+ * 4. Easy way to collapse extra whitespace between button edge and icon.
+ * 5. Make icon a little larger than the parent text size.
+ * 6. Put some space between message and close button.
+ * 7. Removes padding so that element maintains consistent spacing with other
+ *    inner elments
+ */
+
+.c-message__close {
```

Comment:
I would love to see this element's click area be larger, ideally filling the height of the containing element.

Reply:
Great idea, @tylersticka! I agree!

We can also reference the [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/#target-size) in this instance which suggests a minimum of at least 44 by 44 CSS pixels in size. 🙂